dynamic_programming/longest_palindromic_subsequence.py

```diff
@@ -0,0 +1,56 @@
+"""
+Longest Palindromic Subsequence Problem:
+Find the longest sequence of characters from a given input string that
+is the same backwards and forwards
+https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
+"""
+
+
+def longest_palindromic_subsequence(s: str, length: int) -> int:
+
+    """
+    Computes the longest Palindromic sequence of a string
+
+    :param s: str, the string we provide
+    :param length: int, the length of the string
```

Comment:
We can calculate the length of the string with `len(s)` so why require the caller to provide it?